ftw/book/content/book.py

```diff
@@ -84,12 +84,12 @@
 
                 widget=atapi.BooleanWidget(
                     label=_(u'book_label_use_index',
-                            default=u'Embedd subject index')),
-                description=_(u'book_help_use_index',
-                              default=u'When enabled, a keyword index '
-                              'will be included in the PDF.')),
+                            default=u'Embedd subject index'),
+                    description=_(u'book_help_use_index',
+                                  default=u'When enabled, a keyword index '
+                                          'will be included in the PDF.'))),
```

Comment:
If I'm not mistaken, the indentation on this line is missing one space?

Reply:
Oh no, it's correct 👍

Reply:
Yes thats true. Was my editor putting it in line with the unicode start. Changed it though.